tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -588,7 +598,7 @@ def get_screenshot_list(self, node, viewport_size, dpi, page_ranges):
 
 class WdspecExecutor(TestExecutor):
     convert_result = pytest_result_converter
-    protocol_cls = None
+    protocol_cls = None  # type: ClassVar[Type[Protocol]]
```

Comment:
I've probably forgotten something but why aren't we using `    protocol_cls:  ClassVar[Type[Protocol]] = None`?

Reply:
One would have to use `Optional` to allow that. For class variables, for some reason the rules are different when using the comment syntax. I can't find the docs for it now, but you can try it and see :)